src/web/services/simple-modal.service.ts

```diff
@@ -45,6 +46,16 @@ export class SimpleModalService {
         modalRef.componentInstance[key] = value;
       });
     }
+    if (simpleModalOptions?.redirect) {
```

Comment:
- The previous if block already checks for presence `simpleModalOptions`. This block should be moved in there as it can save the `?`.
- Why not combine both `closed` and `dismissed` in the same block?
- This action is not necessarily "redirect"; it just happens that the few usages so far are all redirects. Should name it in a more general way e.g. `onClosed`.